src/jdk.jpackage/macosx/native/applauncher/MacLauncher.cpp

```diff
@@ -52,13 +52,13 @@ void initJvmLauncher() {
     // Create JVM launcher and save in global variable.
     jvmLauncher = AppLauncher()
         .setImageRoot(appImageRoot)
-        .addJvmLibName(_T("Contents/Home/lib/libjli.dylib"))
+        .addJvmLibName(_T("lib/libjli.dylib"))
```

Comment:
the ImageRoot and JvmLibName were previously only used together to get the JvmLibPath - since I am also now using the ImageRoot to see if a runtime is present, I moved the "Contents/Home" from the JvmLibName to the ImageRoot so ImageRoot could be used consistently in platform neutral code.